testing/e2e/testkit.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+function create_git_cached_rule_for_small_repo() {
+  local commit_hash=$1
+  local cache_dir=${2:-"default-dir"}
+
+  # Trim consecutive // characters since Bazel would fail if such exists on a target path
+  local trimmed_path=$(echo ${WORKSPACE_DIR} | tr -s '/')
+
+  if [[ ${cache_dir} == "default-dir" ]]; then
+    cat >${WORKSPACE_DIR}/WORKSPACE <<EOF
+load("//rules/git:git_cached_repository.bzl", "git_cached_repository")
```

Comment:
see my comment about using `wix_build_tools` directly and not copying the rules into this repo.

Reply:
Done, it is now references a `local_repository` i.e.

```
load("@wix_build_tools//rules/git:git_cached_repository.bzl", "git_cached_repository")
```